electron/main.js

```diff
@@ -48,6 +48,16 @@ app.whenReady().then(() => {
         return isNewerVersionAvailable ? `${repository.url}/releases/latest` : undefined;
     });
 
+    // pass current app version to frontend
+    ipcMain.handle('getAppVersion', () => {
+        return version;
+    });
+
+    // pass releases link to frontend
+    ipcMain.handle('getReleasesLink', async () => {
+        return `${repository.url}/releases`;
+    });
```

Comment:
Can you please return them via one object? Or what is the reason to split this into two calls? Do they need to be called separately from each other?